content/en/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md

```diff
@@ -37,7 +37,7 @@ You now need to ensure that the kubectl completion script gets sourced in all yo
 - Add the completion script to the `/etc/bash_completion.d` directory:
 
    ```bash
-   kubectl completion bash >/etc/bash_completion.d/kubectl
+   echo -e "$(kubectl completion bash)" | sudo tee /etc/bash_completion.d/kubectl > /dev/null
```

Comment:
This is better:
```suggestion
   kubectl completion bash | sudo tee /etc/bash_completion.d/kubectl > /dev/null
```




**Ideally**: use https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs to cover the two cases:
1. per user: `kubectl completion bash >>~/.bashrc`
2. system-wide change, editing `/etc/bash_completion.d/kubectl`

and also mention that this is specific to bash; other shells exist.